pyproject.toml

```diff
@@ -0,0 +1,158 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "py-xline"
+dynamic = ["version"]
+description = 'py-xline is an official xline client sdk, written in Python.'
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
+keywords = []
+authors = [
+  { name = "LingKa", email = "cnfty786@gmail.com" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = []
+
+[project.urls]
+Documentation = "https://github.com/unknown/py-xline#readme"
+Issues = "https://github.com/unknown/py-xline/issues"
+Source = "https://github.com/unknown/py-xline"
```

Comment:
Please fix the project url here

Reply:
Oops, sorry I missed that! I've updated it as you recommended.